api/server.go

```diff
@@ -519,6 +519,13 @@ func RunServer(dry bool) http.Handler {
 
 	m.Add("1.0", http.MethodGet, "/plans/routers", AuthorizationRequiredHandler(listRouters))
 
+	m.Add("1.12", http.MethodPost, "/jobs", AuthorizationRequiredHandler(createJob))
+	m.Add("1.12", http.MethodPut, "/jobs/trigger/{name}", AuthorizationRequiredHandler(jobTrigger))
```

Comment:
To be restful like =)
```suggestion
	m.Add("1.12", http.MethodPut, "/jobs/{name}/trigger", AuthorizationRequiredHandler(jobTrigger))
```